pkg/operator/operator.go

```diff
@@ -107,10 +107,13 @@ func RunInstall(cmd *cobra.Command, args []string) {
 	util.KubeCreateSkipExisting(c.NS)
 	util.KubeCreateSkipExisting(c.SA)
 	util.KubeCreateSkipExisting(c.SAEndpoint)
+	util.KubeCreateSkipExisting(c.SAUI)
```

Comment:
I think we can skip this change in the CLI and only have it in the OLM

Reply:
✅